build.sbt

```diff
@@ -6,20 +6,21 @@ import scala.concurrent.duration.DurationInt
 
 // common settings (apply to all projects)
 ThisBuild / organization := "com.gu"
-ThisBuild / version := "0.4.2"
+ThisBuild / version := "0.5.0"
 ThisBuild / scalaVersion := "2.13.10"
 ThisBuild / scalacOptions ++= Seq("-deprecation", "-feature", "-unchecked", "-Xfatal-warnings")
 
+// We need to avoid a binary-incompatible version conflict - because janus-config-tools v.0.0.5 requires com.typesafe.play:twirl-api_2.13:1.4.2
```

Comment:
Perhaps we can remove this line once we bump janus-config-tools to 0.0.6, which we can do straight after this.